backend/installation/README.md

```diff
@@ -1,55 +1,98 @@
 # Backend Installation
 
-{% hint style="warning" %}
-This documentation should be split into a **local** and a **docker** installation variant. Also maybe there should be a main docker installation guide for the whole system at once!?
-{% endhint %}
 
-### Clone Repository
-
-The Backend Repository can be found on github.  
+The Backend repository can be found on GitHub.
 [https://github.com/Human-Connection/Nitro-Backend](https://github.com/Human-Connection/Nitro-Backend)
 
-## Fork the repo
+{% hint style="info" %} TODO: Create documentation section for How to Start and Beginners. {% endhint %}
+
+Here are some general informations about our [GitHub Standard Fork & Pull Request Workflow](https://gist.github.com/Chaser324/ce0505fbed06b947d962).
+
+### Fork the Repository
 
 Click on the fork button.
 
-![Fork screenshot](https://i.imgur.com/syoN9rF.png)
+![Fork screenshot](../../.gitbook/assets/screenshot-forking-nitro-backend.png)
+
+### Clone your new Repository
 
-After you have forked, modify the following command to add your Github user name.
+Set the current working folder to the path in which the backend repository should be cloned (copied).
+
+```bash
+$ cd PATH-FOR-NITRO-BACKEND
+```
+
+For cloning your new repository to your local machine modify the following command to add your GitHub user name.
 
 {% tabs %}
 {% tab title="HTTPS" %}
 ```bash
-git clone https://github.com/YOUR-GITHUB-USERNAME/Nitro-Backend.git
+$ git clone https://github.com/YOUR-GITHUB-USERNAME/Nitro-Backend.git
+
```

Comment:
*now* we have to add `cd Nitro-Backend` between the two commands :laughing:

Reply:
@roschaefer Sorry, I was concentrated. Hope I got it right now. ❤️